### ISSUE TITLE
Updating the 'Poverty' tab indicators on the Global Picture

### ISSUE THREAD

**Issue by Duncan-Knox** [devinit/datahub#336]

**Comment by Duncan-Knox** [comment]
New data not reflected on staging. Data resubmitted to @Napho via email for upload to database at a convenient time.

**Comment by edwinmp** [comment]
@Duncan-Knox Maybe there's something I need to do on my end. Could you get me details on the changes required here; what indicators were updated & what the new change should be. Even if it's just for one indicator, can use that to confirm whether the data is pulling through from the database or not (if it's there).

**Comment by Duncan-Knox** [comment]
@edwinmp I'm unsure.  Here are the data series names for the Global Picture poverty tab, one difference is that these new submissions have data for Cambodia (DI_ID: KH) removed whereas we noticed Cambodia was still showing data on staging, suggestion the DB still contain older data:
--
data_series.depth_of_extreme_poverty_190
data_series.2013_p20_population
data_series.percent_in_p20_national
data_series.poverty_310
data_series.poverty_190
data_series.poor_people_190

Happy to chat about this tomorrow or at a convenient time.

**Comment by edwinmp** [comment]
Hi @Napho, still can't see the changes above pulling through on staging. Any update on this?

**Comment by Duncan-Knox** [comment]
Seconded @edwinmp cannot see the changes for the 6 indicators submitted. Happy to go into the details of this to see whether all 6 can be added live to staging.

**Comment by edwinmp** [comment]
Hi @Duncan-Knox ... I think these should be coming through now. Could you please confirm.
Thanks.

**Comment by Duncan-Knox** [comment]
Morning @edwinmp, I can confirm the following 4 are coming through:

data_series.depth_of_extreme_poverty_190
data_series.2013_p20_population
data_series.percent_in_p20_national
data_series.poverty_310

And I was due to discuss:

data_series.poverty_190
data_series.poor_people_190

With @Napho

**Comment by Duncan-Knox** [comment]
I had a separate query on these @edwinmp I was wondering how complex it would be to remove certain years from showing across some of these specific indicators on the poverty tab. This would be a case of years to show data for being provided and others being removed from the slider. e.g. show 2000, 2003, 2006, 2011, 2012, 2013, etc. Is this a time consuming fix?

**Comment by edwinmp** [comment]
Will have a look into it and get back to you. You should create a feature request for it though :)

**Comment by Duncan-Knox** [comment]
@edwinmp thanks, will do.

**Comment by Duncan-Knox** [comment]
Blocked until 

data_series.poverty_190
data_series.poor_people_190

Discussed with  @Napho

**Comment by akmiller01** [comment]
Hey @Duncan-Knox , is this still blocked? Just spoke with Zach and he said he sent you a correction.

**Comment by Duncan-Knox** [comment]
Hi @akmiller01 I have the file from Zach available, however it needs to be added into the format I submitted to @Napho in the first place. Are you please able to help out here? This however is a separate issue to why this file is blocked. 

@Napho would be better able to provide information on the reason why these have been blocked. Which related to the following data series:

data_series.2013_p20_population
data_series.percent_in_p20_national

I am unsure if this has been resolved.

**Comment by Duncan-Knox** [comment]
Following on from what is noted above it is worth noting this issue has taken an extension with the data_series.poor_people_190 needing resubmission.

**Comment by k8hughes** [comment]
Data is in the drop box folder so this should be ready to work on.

**Comment by Napho** [comment]
@edwinmp  data_series.2013_p20_population now has data for all these years not just 2013

 2013
 2012
 2011
 2010
 2008
 2005
 2002
 1999
 1996
 1993
 1990
 1987
 1984
 1981

**Comment by edwinmp** [comment]
@Napho Updated the years above. Is there anything else pending on this task?

**Comment by Duncan-Knox** [comment]
Will need to re-submit Depth of extreme poverty (poverty gap) data which carries the decimal/percent issue. Have done this now on drop box and will inform @Napho. This should be the final action for these.

**Comment by Duncan-Knox** [comment]
Checked and fine. Excellent.

**Comment by Duncan-Knox** [comment]
Redone and sign-off in staging for new PovcalNet October data. Visuals altered to show data up to 2015.
